### ISSUE TITLE
Disease Prediction using Decision Tree...How it works?

### ISSUE THREAD

**Issue by Hafsa469** [itachi9604/healthcare-chatbot#3]
Hi! Can you please tell me the basic working of a tree that on which bases it is predicting the diseases because in mostly cases the irrelevant diseases are predicted?
Like for high_fever it is pedicting Arthritis while high fever is not in its symptoms

**Comment by Siddharth-L** [comment]
yes the predictions are irrelevant and so working of tree can be explianed !!
